scripts/build_backend.py

```diff
@@ -85,11 +85,13 @@
             "-DDPCPP_INSTALL_DIR=" + DPCPP_ROOT,
             "-DCMAKE_C_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "clang"),
             "-DCMAKE_CXX_COMPILER:PATH=" + os.path.join(DPCPP_ROOT, "bin", "dpcpp"),
+            "-DDPCTL_BUILD_CAPI_TESTS=ON",
             "-DDPCTL_ENABLE_LO_PROGRAM_CREATION=ON",
             backends,
         ]
         subprocess.check_call(cmake_args, stderr=subprocess.STDOUT, shell=False)
         subprocess.check_call(["make", "V=1", "-j", "4"])
+        subprocess.check_call(["make", "check"])
```

Comment:
Just a note: `check_call` raises an exception if the subprocess returns with a non-zero return code, so any failures in C-API test suite will fail the Python build as desired.